certbot/cli.py

```diff
@@ -357,7 +357,8 @@ def _get_help_string(self, action):
     }),
     ("delete", {
         "short": "Clean up all files related to a certificate",
-        "opts": "Options for deleting a certificate"
+        "opts": "Options for deleting a certificate",
+        "usage": "\n\n  certbot delete --cert-name CERTNAME"
```

Comment:
I'm not sure of the reasoning, but it seems that the existing `usage` lines begin _and_ end with `\n\n`. Perhaps the new ones should as well?